### DISCUSSION TITLE
Date Types for the graphql-kotlin

### DISCUSSION THREAD

**Discussion by rajkumar1979** [ExpediaGroup/graphql-kotlin#771] [Q&A] [answered]
what are the valid date types for graphql-kotlin? local date time and zoned date time are not supported

**Comment by smyrick** [comment, marked as answer]
@rajkumar1979 I have updated the docs in #772 to make this a little more clear, but you can also search our issues for previously asked questions if you are having some basic setup problems on your project

https://github.com/ExpediaGroup/graphql-kotlin/issues?q=datetime
